layouts/layouts.module.css

```
@@ -215,6 +215,8 @@
     main {
       @apply max-w-[660px]
         gap-4;
+
+      word-break: break-all;
```

```suggestion
        gap-4
        break-all;

```

You can use `break-all` https://tailwindcss.com/docs/word-break